packages/docs/source/base/spacing.md

```diff
@@ -2,11 +2,12 @@
 
 
 <ul class="sample--list">
-  <li class="sample--spacing-huge-spacing"></li>
-  <li class="sample--spacing-big-spacing"></li>
-  <li class="sample--spacing-base-spacing"></li>
-  <li class="sample--spacing-small-spacing"></li>
-  <li class="sample--spacing-tiny-spacing"></li>
+  <li class="sample--spacing-spacing-xxl"></li>
```

Comment:
This enumeration feels so weird to me, especially since they aren't symmetrical. 

Do we have use cases for all of these? If not, should we even surface them yet?

e.g. do we want product designers using XL spacing? are there valid uses for that?

Reply:
this is solely for documentation purposes and is a function of search and replace changing them is pretty straight forward. I can fix this.

Reply:
This was resolved, we updated the spacing and as a result xxl is no longer in use. Still, I am no longer a fan of t-shirt enumeration coming out of this admin restyle project. We should re-evaluate this soon.